### ISSUE TITLE
TabView: Hides menuItems items except the first one under actions facet of TabView

### ISSUE THREAD

**Issue by aripddev** [primefaces/primefaces#12338]
### Describe the bug

When put the menu button to the actions facet of TabView, only the first one is shown.

```xhtml
<p:tabView>
    <f:facet name="actions">
        <p:menuButton value="Menu button">
            <p:menuitem value="menu item 1"/>
            <p:menuitem value="menu item 2"/>
            <p:menuitem value="menu item 3"/>
            <p:menuitem value="menu item 4"/>
        </p:menuButton>
    </f:facet>
    <p:tab title="Tab1"/>
    <p:tab title="Tab2"/>
    <p:tab title="Tab3"/>
</p:tabView>
```

<img width="1680" alt="Screenshot 2024-07-29 at 16 15 28" src="https://github.com/user-attachments/assets/a92c6783-7018-46bc-8407-fb96c87f99f9">

### Reproducer

[primefaces-test.zip](https://github.com/user-attachments/files/16413773/primefaces-test.zip)

### Expected behavior

all menu items must be shown

### PrimeFaces edition

Community

### PrimeFaces version

15.0.0-SNAPSHOT

### JSF implementation

All

### JSF version

2.3

### Java version

17

**Comment by melloware** [comment]
This is really weird.  all 4 menu items are in the HTML

**Comment by melloware** [comment]
ok the menu is rendered like this:
![image](https://github.com/user-attachments/assets/8639daa0-541e-4649-b6b7-98c9936b4964)

I think it has to do with Tab `<LI>` and Menu `<LI>` and both trying to `float` css.

**Comment by melloware** [comment]
Try this CSS:

```css
.ui-tabs .ui-tabs-nav li .ui-menu-list {
  white-space: initial;
}
```

**Comment by aripddev** [comment]
Yes, now it's fixed. Thanks

**Comment by melloware** [comment]
Nice I will fix this for 14.0.4
